src/Umbraco.Web.UI.Client/src/common/directives/components/umbnestedcontent.directive.js

```diff
@@ -49,6 +49,9 @@
 
                         }
                     });
+                    
+                    // Some property editors need to performe an action after all property editors have reacted to the formSubmitting.
+                    $scope.$broadcast("postFormSubmitting", { scope: $scope });
```

Comment:
I actually choose to remove it from this spot. The event is still broadcasted from the Formhelper, so there is no reason to do an extra broadcast here. The formSubmitting broadcast further up is a hack to make sub-editor sync before this takes place. Not preferable, but that we have to stick with.

To avoid testing wether BlockList property editor work inside Nested Content I have made things easier by dis-allowed Umbraco.BlockList from NestedContent.